cmd/guacone/cmd/known.go

```diff
@@ -87,10 +90,11 @@ var queryKnownCmd = &cobra.Command{
   <subject> is in the form of "<purl>" for package, "<vcs_tool>+<transport>" for source, or "<algorithm>:<digest>" for artiact.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := logging.WithLogger(context.Background())
-		logger := logging.FromContext(ctx)
+		//logger := logging.FromContext(ctx)
```

Comment:
logger commented out?

Reply:
removed.

wasn't used in top level since most of the code abstracted out to the helper function where logger is obtained directly.